build/common/config/.golangci.yml

```
@@ -164,13 +180,32 @@ issues:
   # excluded by default patterns execute `golangci-lint run --help`
   exclude:
     - composite literal uses unkeyed fields
+    - return statements should not be cuddled if block has more than two lines
+    - declarations should never be cuddled
+    - don't use leading k in Go names
 
   exclude-rules:
-    # Exclude some linters from running on test files.
-    - path: _test\.go$|^tests/|^samples/
+    # Allow dot imports in the tests.
+    - path: _test\.go$|^test/
+      linters:
+        - gci
+        - golint
+        - revive
+        - stylecheck
+      source: \. "github\.com/onsi/(gomega|ginkgo)"
```

Other repos have it like this. I'm not sure if that was missed in the merge conflicts.
```suggestion
      source: \. "github\.com/onsi/(gomega|ginkgo/v2)"
```

---

I've updated this.  Thanks!